src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -62,10 +65,16 @@ export class AddBulkConcepts extends Component {
     const { id } = e.target;
     this.setState({ currentPage: Number(id) });
   };
+
+  filterCaseInsensitive = (filter, rows) => {
```

Comment:
What does `filterCaseInsensitive` mean @dessHub?

Reply:
@dondrzzy It means it does filter based on the given input regardless of the input case. For instance, `Random` will match `random` and `Random`.